src/nvim/version.c

```diff
@@ -2709,7 +2709,7 @@ void intro_message(int colon)
   static char *(lines[]) = {
     N_(NVIM_VERSION_LONG),
     "",
-    N_("by Bram Moolenaar et al."),
+    N_("by al."),
```

Comment:
`by al.` is indeed strange.  I don't think that's going to make sense outside of the context of it previously being `et al.`.  What about either

* > by Neovim contributors
* > based on Vim